src/components/pages/AccountDashboard/styles/AccountDashboard.css

```diff
@@ -1,66 +1,236 @@
+    transform: translateY(0px);
+    opacity: 1;
+  }
+}
+
+.profile {
+  -webkit-animation: popUp ease-in-out 350ms;
+  animation: popUp ease-in-out 350ms;
   background: #bdbdbd;
-  min-height: 100vh;
+  border-radius: 25px;
+  box-shadow: 0 0 40px 0px rgba(0, 0, 0, 0.17);
+  margin-top: 40px;
+  margin-left: -250px;
+  left: 50%;
+  position: absolute;
+  width: 500px;
+}
+.profile__account {
+  align-self: center;
   display: -webkit-box;
-  display: -ms-flexbox;
   display: flex;
-  font-weight: 400;
-  font-family: "Fira Sans", sans-serif;
-  padding: 5%;
+  -webkit-box-flex: 1;
+  flex: 1;
+  -webkit-box-pack: end;
+  justify-content: flex-end;
+  padding-right: 115px;
 }
 
-h1,
-h2,
-h3,
-h4,
-h5,
-h6,
-label,
-span {
-  font-weight: 500;
-  font-family: "Fira Sans", sans-serif;
+.profile__header {
+  display: -webkit-box;
+  display: flex;
+  margin-bottom: 20px;
+}
+.profile__icon {
+  -webkit-box-flex: 0;
+  flex: none;
+  font-size: 1.5em;
+  margin-right: 10px;
+  padding-top: 3px;
+}
+.profile__icon--green {
+  color: #009689;
 }
 
-body,
-html,
-.App,
-#root,
-.auth-wrapper {
+.profile__icon--red {
+  color: #e61d1d;
+}
+.profile__icon--blue {
+  color: #2196f3;
+}
+.profile__key {
+  font-family: "Fira Sans", sans-serif;
+  font-size: 23px;
+  font-weight: 400;
+  text-align: center;
+}
+.profile__picture {
+  background: #ffffff;
+  border-radius: 100px;
+  border: 10px solid #ffffff;
+  height: 125px;
+  position: absolute;
+  left: 10%;
+  top: -50px;
+  width: 125px;
+}
+.profile__picture:before {
+  border-radius: 100px;
+  box-shadow: 0 0 40px 0px rgba(0, 0, 0, 0.17);
+  content: "";
+  height: calc(100% + 20px);
+  left: -10px;
+  position: absolute;
+  top: -10px;
+  width: calc(100% + 20px);
+  z-index: -1;
+}
+.profile__picture img {
+  border-radius: 100px;
+  height: 100%;
   width: 100%;
 }
-
-.auth-wrapper {
+.profile__stat {
+  -webkit-animation: slideUp ease-in-out 350ms forwards;
+  animation: slideUp ease-in-out 350ms forwards;
+  border-right: 1px solid #e9e9e9;
   display: -webkit-box;
-  display: -ms-flexbox;
   display: flex;
+  -webkit-box-flex: 1;
+  flex: 1;
   -webkit-box-pack: center;
-      -ms-flex-pack: center;
-          justify-content: center;
-  -webkit-box-orient: vertical;
-  -webkit-box-direction: normal;
-      -ms-flex-direction: column;
-          flex-direction: column;
-  text-align: left;
+  justify-content: center;
+  opacity: 0;
+  padding: 20px 8px;
+  -webkit-transform: translateY(5px);
+  transform: translateY(5px);
+}
+.profile__stat:last-of-type {
+  border-right: none;
+}
+.profile__stat:nth-child(1) {
+  -webkit-animation-delay: 400ms;
+  animation-delay: 400ms;
+}
+.profile__stat:nth-child(2) {
+  -webkit-animation-delay: 500ms;
+  animation-delay: 500ms;
+}
+.profile__stat:nth-child(3) {
+  -webkit-animation-delay: 600ms;
+  animation-delay: 600ms;
+}
+.profile__stats {
+  display: -webkit-box;
+  display: flex;
+}
+.profile__username {
+  font-family: "Fira Sans", sans-serif;
```

Comment:
Can we just define font-family to the whole component once instead of duplicate times?